soccer/Robot.cpp

```diff
@@ -196,28 +191,30 @@ void OurRobot::move(Geometry2d::Point goal, Geometry2d::Point endVelocity) {
              << goal.y() << ")" << std::endl;
     }
 
-    intent().motion_command = std::make_unique<Planning::PathTargetCommand>(
-        MotionInstant(goal, endVelocity));
+    RobotInstant goal_instant;
+    goal_instant.pose = Pose{goal, 0};
```

Comment:
This sets the heading target to 0 for every normal move?

Reply:
Heading for this was ignored. I'm changing it to a linear motion instant to represent that.